content/docs/2.9/operate/cluster.md

```diff
@@ -130,3 +130,29 @@ To specify values other than their defaults, you can set the following environme
 | KEDA_METRICS_LEADER_ELECTION_LEASE_DURATION  | Metrics Server | LeaseDuration    |
 | KEDA_METRICS_LEADER_ELECTION_RENEW_DEADLINE  | Metrics Server | RenewDeadline    |
 | KEDA_METRICS_LEADER_ELECTION_RETRY_PERIOD    | Metrics Server | RetryPeriod      |
+
+## Restrict Secret Access
+
+By default, KEDA requires adding `secrets` to the cluster role as following:
+```yaml
+- apiGroups:
+  - ""
+  resources:
+  - external
+  - pods
+  - secrets
+  - services
+  verbs:
+  - get
+  - list
+  - watch
+```
+However, this might lead to security risk (especially in production environment) since it will grant permission to read `secrets` from all namespaces.
+
+To restrict `secret` access and limited to KEDA namespace, you could add `KEDA_RESTRICT_SECRET_ACCESS` as environment variable to both KEDA Operator and KEDA Metrics Server:
+```yaml
+env:
+  - name: KEDA_RESTRICT_SECRET_ACCESS
+    value: "true"
+```
+This allows you to omit `secrets` from the cluster role which will disallow `TriggerAuthentication` to be used for your triggers. You can, however, still use `ClusterTriggerAuthentication`.
```

Comment:
Will any `TriggerAuthentication` or only whose use secrets? I mean, will a `TriggerAuthentication` with a pod identity fail?

Reply:
Yes, I will update this